projects/canopy/src/lib/progress-indicator/progress-indicator.component.ts

```diff
@@ -0,0 +1,33 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  HostBinding,
+  Input,
+  ViewEncapsulation,
+} from '@angular/core';
+
+let nextUniqueId = 0;
+
+@Component({
+  selector: 'lg-progress-indicator',
+  templateUrl: './progress-indicator.component.html',
+  styleUrls: [ './progress-indicator.component.scss' ],
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class LgProgressIndicatorComponent {
+  @HostBinding('class.lg-progress-indicator') class = true;
+  @Input() max = 0;
+  @Input() value = 0;
+  @Input() showProgressBar = true;
+  @Input() showAsPercentage = false;
```

Comment:
I think this is a bit of a janky name; it implies a preferred option which I'm not sure it should. How about calling it something like `displayAs` and supporting two options: `'percentage' | 'step'`

Reply:
This nice and leaves the door open for more options in future 👍 

Could have an associated type, e.g.

```
export type ProgressJourneyDisplayFormat = 'percentage' | 'step'
```